src/frontends/onnx/frontend/src/core/graph.cpp

```diff
@@ -25,8 +25,9 @@
 
 using namespace ov;
```

Comment:
should we remove such `using namespace ov` then? Because all content is already in the root `ov` namespace

Reply:
In theory - yes, I'll check on a local build